pkgs/phps.nix

```diff
@@ -43,9 +54,14 @@ let
 
               "--enable-libxml"
               "--with-libxml-dir=${prev.libxml2.dev}"
+              (
+                if prev.lib.versionAtLeast args.version "7.3"
+                then "--with-pcre-regex=${prev.pcre2.dev}"
```

Comment:
7.3 should actually use pkg-config so I would not expect this to be necessary:

https://github.com/php/php-src/blob/php-7.3.33/ext/pcre/config0.m4#L20

Reply:
No, when I build without the flag, it didn't build for php7.3.
Actually, the original [generic.nix (for nixos-21.05)](https://github.com/NixOS/nixpkgs/blob/nixos-21.05/pkgs/development/interpreters/php/generic.nix#L179) also had the pcre2 line. But after deprecating php7.3, it [has disappeared](https://github.com/NixOS/nixpkgs/blob/nixos-22.05/pkgs/development/interpreters/php/generic.nix#L226).